package.json

```diff
@@ -39,7 +39,7 @@
     "json2yaml": "^1.1.0",
     "jsonwebtoken": "^8.5.1",
     "koncorde": "^1.2.1",
-    "kuzzle-common-objects": "^3.0.14",
+    "kuzzle-common-objects": "git://github.com/kuzzleio/kuzzle-common-objects.git#3-dev",
```

Comment:
You cannot leave this reference in `package.json`. We need to release kuzzle-common-objects before this PR can be merged.

Reply:
Right! Thanks